content/en/blog/2024/gateway-mesh-ga/index.md

```diff
@@ -0,0 +1,98 @@
+attribution: John Howard - solo.io
+keywords: [istio, traffic, API]
+target_release: 1.22
+---
+
+We are thrilled to announce that Service Mesh support for the [Gateway API](https://gateway-api.sigs.k8s.io/) is now official "Stable"!
+With this release (part of Gateway API v1.1 and Istio v1.22), users can make use of the next-generation traffic management APIs for both ingress ("north-south") and service mesh use cases ("east-west").
+
+## What *is* the Gateway API?
+
+The Gateway API is a collection of APIs that are part of Kubernetes, focusing on traffic routing and management.
+The APIs are inspired by, and serve many of the same roles as, Kubernetes' `Ingress`, and Istio's `VirtualService` and `Gateway` APIs.
+
+These APIs have been under development both in Istio, as well as with [broad collaboration](https://gateway-api.sigs.k8s.io/implementations/), since 2020, and have come a long way since then.
+While the API initially targeted only serving ingress use cases (which went GA [last year](https://kubernetes.io/blog/2023/10/31/gateway-api-ga/), we had always envisioned allowing the same APIs to be used for traffic *within* a cluster as well.
+
+With this release, that vision is made a reality: Istio users can use the same routing API for all of their traffic!
+
+## Getting started
+
+Throughout the Istio documentation, all of our examples have been updated to show how to use the Gateway API, so explore some of the [tasks](/docs/tasks/traffic-management/) to gain a deeper understanding.
+
+Using Gateway API for service mesh should feel familiar both to users already using Gateway API for ingress, and users using `VirtualService` for service mesh today.
+
+* Compared to Gateway API for ingress, the routes target a `Service` instead of a `Gateway`.
+* Compared to VirtualService, where routes associate with a set of `hosts`, routes target a `Service`.
+
+Here is a simple example, which showcases routing requests to two different versions of a `Service` based on the request.
+
+{{< text yaml >}}
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: reviews
+spec:
+  parentRefs:
+  - group: ""
+    kind: Service
+    name: reviews
+    port: 9080
+  rules:
+  - matches:
+    - headers:
+      - name: my-favorite-service-mesh
+        value: istio
+    filters:
+    - type: RequestHeaderModifier
+      requestHeaderModifier:
+      add:
+        - name: hello
+          value: world
+    backendRefs:
+    - name: reviews-v2
+      port: 9080
+  - backendRefs:
+    - name: reviews-v1
+      port: 9080
+{{< /text >}}
+
+Breaking this down, we have a few parts:
+* First, we identify what routes we should match.
+  By attaching our route to the `reviews` `Service`, we will apply this routing configuration to all requests that were originally targeting `reviews`.
+* Next, we can (optionally) have a variety of matches on the request if we want to behave differently based on the request. This is handled by the `matches`.
```

Comment:
```suggestion
* Next, we can (optionally) have a variety of matches on the request, if we want to behave differently based on the request. This is handled by the `matches`.
```

(Can you have a go at rewriting this?  It doesn't make sense/repeats 'matches' and 'request' too much for my liking)